modules/ROOT/pages/7.1-release-notes.adoc

```diff
@@ -109,10 +109,14 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 {productname} 7.1 also includes the following improvement<s>:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== In the custom view, the scrollbar of the container were not visible if its height was greater than the editor.
```

Comment:
This title doesn't encapsulate all the changes, and it seems there are multiple changie entries 
associated with this ticket. Maybe a rewording of [this changie entry](https://github.com/tinymce/tinymce-premium/pull/152/files) could be better?
```suggestion
=== Improved the responsiveness of the Custom view on small screens
```

Reply:
@tiny-ben-tran please advise on what we want to do here, if docs change the `title` i would expect that the `.changie.yml` is also updated to ensure it matches the `changelog` for tinymce.